<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2D RBF Interpolator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="VisualizingWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_x1">
          <property name="text">
           <string>x1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_x1"/>
        </item>
        <item>
         <widget class="QLabel" name="label_x2">
          <property name="text">
           <string>x2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_x2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_y">
          <property name="text">
           <string>y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_y"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_export">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_import">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VisualizingWidget</class>
   <extends>QWidget</extends>
   <header>visualizingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
